api/edge_api/identities/traits/views.py

```diff
@@ -0,0 +1,78 @@
+import marshmallow
+from drf_yasg2.utils import swagger_auto_schema
+from flag_engine.api.schemas import APITraitSchema
+from flag_engine.identities.builders import build_identity_dict
+from rest_framework import status, viewsets
+from rest_framework.exceptions import NotFound, ValidationError
+from rest_framework.response import Response
+
+from edge_api.identities.views_mixins import GetIdentityMixin
+from environments.identities.models import Identity
+
+from .permissions import EnvironmentIdentityTraitsPermission
+from .serializers import EdgeIdentityTraitsResponseSerializer
+
+trait_schema = APITraitSchema()
+
+
+class EdgeIdentityTraitsViewSet(viewsets.ModelViewSet, GetIdentityMixin):
```

Comment:
Does this need to be a `ModelViewSet`? What benefits do we get from it since we're not actually operating on a model?